### ISSUE TITLE
System.NullReferenceException on context.BulkUpdate

### ISSUE THREAD

**Issue by rakewell** [borisdj/EFCore.BulkExtensions#1053]
May be related to #1040

```

var proxy = context.ProxyServers.Where(ps => ps.ProxyServerId == request.ExchangeAccountDataImportRequestProxyServerId).FirstOrDefault();

//disable the proxy
proxy.ProxyServerEnabled = false;
proxy.ProxyServerLastError = $"{proxyEx.Message}_{proxyEx.SourceExceptionMessage}";
context.BulkUpdate( new List<ProxyServers> { proxy }); //exception thrown here

```
```

System.NullReferenceException: 'Object reference not set to an instance of an object.'
   at EFCore.BulkExtensions.TableInfo.LoadData[T](DbContext context, Type type, IList`1 entities, Boolean loadOnlyPKColumn)
   at EFCore.BulkExtensions.TableInfo.CreateInstance[T](DbContext context, Type type, IList`1 entities, OperationType operationType, BulkConfig bulkConfig)
   at EFCore.BulkExtensions.DbContextBulkTransaction.Execute[T](DbContext context, Type type, IList`1 entities, OperationType operationType, BulkConfig bulkConfig, Action`1 progress)
   at EFCore.BulkExtensions.DbContextBulkExtensions.BulkUpdate[T](DbContext context, IList`1 entities, BulkConfig bulkConfig, Action`1 progress, Type type)
   at CoinTradeLedger.WebUi.Services.ExchangeAccountDataRequestProcessor.<Execute>d__39.MoveNext() in F:\Data\Dev\Products\CoinTradeLedger\Src\CoinTradeLedger.WebUi\Services\ExchangeAccountDataRequestProcessor.cs:line 331

```

**Comment by rakewell** [comment]
context.BulkUpdate( new List<ProxyServers> { proxy });

**Comment by rakewell** [comment]
"
 context.BulkUpdate( new List<ProxyServers> { proxy });
"

**Comment by edokan** [comment]
duplicate of #1040, fixed in #1039
